.env.compose

```
@@ -538,6 +538,7 @@ MCP_SERVER_MODE=stdio
 
 # HTTP Transport Settings
 MCP_HTTP_PORT=3001
+MCP_AUTH_PORT=3003
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_AUTH_PORT key should go before the MCP_HTTP_PORT key

(UnorderedKey)